recipe/meta.yaml

```diff
@@ -26,15 +26,14 @@ requirements:
     - setuptools
   run:
     - python
-    - ipykernel >=4.1  # not a real dependency, but require the reference kernel
+    - traitlets
     - ipython_genutils
-    - jupyter_client >=4.1
     - jupyter_core
+    - jupyter_client >=4.1
     - pygments
-    - pyqt
-    - pyzmq >=17.1
```

Comment:
`pyzmq` is required https://github.com/jupyter/qtconsole/blob/7c739f4a9e7a1218c58a52374598b9d9202dd0c1/setup.py#L76

Reply:
Thanks Serhii! added back to run section :)